docs/mine/lotus/split-markets-miners.md

```diff
@@ -122,29 +189,31 @@ lotus-miner --markets-repo=~/.lotusmarkets init service --type=markets \
                                                         /tmp/backup.cbor
 ```
 
-3. Optionally update your miner's `multiaddr` on-chain - in case your `markets` instance is publicly exposed at a different location compared to your existing monolith miner, you also need to update your `multiaddr` on-chain and advertise the correct one to clients:
+3. If during this procedure, you ended up changing the `multiaddr` of your `markets`
+process to a different host and/or port, you must update miner's `multiaddr` on-chain.
+Otherwise, clients wishing to make deals with you will no longer be able to connect to
+your node.
 
 ```shell
 lotus-miner actor set-addrs <NEW_MULTIADDR>
 ```
 
-### 4. Move the DAG store directory to the markets node repository
+#### 5. Move the DAG store directory to the markets node repository (optional)
+
+If you are running a lotus version with a DAG store, you can optionally move the
+DAG store directory to the lotus markets repository, to avoid having to reindex
+all storage deals when the markets node starts up.
```

Comment:
Might include a comment on transferring the content of the directory, if markets node is going to be running on a separate machine.